docker/build

```diff
@@ -1,22 +1,9 @@
+#!/bin/sh
 set -xeu
 
-echo "-- ${DMD} -- "
-
-# Deduce dmd package name/version
-. .circleci/dmd.sh
-
-# Update package cache
-apt-get update
-
-if [ "`echo $DMD_PKG | grep dmd-compiler`" ]
-then
-    # Remove pre-installed DMD package to avoid conflicts
-    apt-get remove -y dmd-bin
-fi
-
 # Install dependencies
 apt-get install -y \
-    $DMD_PKG \
+    d1to2fix=0.10.* \
```

Comment:
Why is this necessary?

Reply:
I copied current ocean build script blindly, nothing is intentional :)

Reply:
ping